app/app.rb

```diff
@@ -194,6 +194,11 @@ class App < Sinatra::Base
 		get "/:site/files/?*" do
 			file = site.file(params['splat'].join('/'))
 
+			if params[:download] && file.exist? && !file.directory?
+				authorize file, :show?
+				return send_file file.path, filename: file.name, type: "Application/octet-stream"
+			end
```

Comment:
Should use attachment instead of octet-stream content type to force download (and provide filename for download)